stripe/src/main/java/com/stripe/android/payments/StripeBrowserLauncherViewModel.kt

```diff
@@ -3,19 +3,62 @@ package com.stripe.android.payments
 import android.app.Application
 import android.content.Intent
 import android.net.Uri
+import androidx.browser.customtabs.CustomTabColorSchemeParams
+import androidx.browser.customtabs.CustomTabsIntent
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
 import com.stripe.android.AnalyticsEvent
 import com.stripe.android.PaymentConfiguration
+import com.stripe.android.R
 import com.stripe.android.auth.PaymentBrowserAuthContract
 import com.stripe.android.networking.AnalyticsRequestExecutor
 import com.stripe.android.networking.AnalyticsRequestFactory
 
 internal class StripeBrowserLauncherViewModel(
     private val analyticsRequestExecutor: AnalyticsRequestExecutor,
-    private val analyticsRequestFactory: AnalyticsRequestFactory
+    private val analyticsRequestFactory: AnalyticsRequestFactory,
+    private val browserCapabilities: BrowserCapabilities,
+    private val intentChooserTitle: String
 ) : ViewModel() {
```

Comment:
If you extend AndroidViewModel instead of ViewModel, you can get the application context directly.

Reply:
I considered using `AndroidViewModel` but I realized we only need it for a single string so it seemed like overkill.

Reply:
ok, I needed it for two strings in another viewModel.   We should decide how we want to handle this type of thing and then just be consistent.  I think it is fine for this review though.

Reply:
Yes, agreed! I believe we're taking a different approach in one of the PaymentSheet view models for strings.